retroshare.pri

```diff
@@ -118,9 +118,11 @@ CONFIG *= rs_gxs_trans
 
 # To enable GXS based async chat append the following assignation to qmake
 # command line "CONFIG+=rs_async_chat"
-CONFIG *= no_rs_async_chat
+#CONFIG *= no_rs_async_chat
+CONFIG+=rs_async_chat
```

Comment:
rs_async_chat should be left disabled by default. How can adding additional code help compilation anyway?